scripts/provisioning/s3_start

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+import sys
+import argparse
+import traceback
+import errno
+import logging
+import os
+import socket
+from logging import handlers
+
+#Logger details
+s3_single_entry_logger_name = \
+    "s3-deployment-logger-" + "[" + str(socket.gethostname()) + "]"
+s3_single_entry_log_file = \
+    "/var/log/seagate/s3/s3deployment/s3_single_entry.log"
+s3_single_entry_log_directory = "/var/log/seagate/s3/s3deployment/"
+s3_single_entry_log_format = \
+    "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+
+def main():
+
+  parser = argparse.ArgumentParser("S3server single entry command")
+  parser.add_argument("--service", required=True)
+
+  args = parser.parse_args()
+
+  create_logger_directory()
+  logger = create_logger()
+
+  try:
+    if args.service == 's3server':
+      # Action
+    elif args.service == 's3authserver':
+      os.system("sh /opt/seagate/cortx/auth/startauth.sh")
+    elif args.service == 's3bgproducer':
+      # Action
+    elif args.service == 's3bgconsumer':
+      # Action
```

Comment:
add 'else' here to handle wrong/unsupported argument (or define specific accepted list in the arg parser).

Reply:
Done.